tests/Unit/ViewHelpers/Fixtures/UserWithoutToString.php

```diff
@@ -40,4 +40,12 @@ public function isNamed()
     {
         return !empty($this->name);
     }
+
+    /**
+     * @return boolean
+     */
+    public function hasNamed()
```

Comment:
I think we should use another name here to make sure we are testing both the `has` and `is` prefixes, for example `hasVirtualProperty` then test property `user.virtualProperty` to determine if the `has` prefix is being detected and used correctly.